neon_core/__init__.py

```diff
@@ -55,25 +62,23 @@ def setup_holmes_config():
             "neon_enclosure": "neon_core"
         }
     }
+
     cfg = {}
-    if exists(HOLMES_CONFIG):
-        try:
-            with open(HOLMES_CONFIG) as f:
-                cfg = json.load(f)
-        except:
-            pass
-    cfg = merge_dict(cfg, _NEON_HOLMES_CONFIG)
-    with open(HOLMES_CONFIG, "w") as f:
-        json.dump(cfg, f, indent=4, ensure_ascii=True)
+    try:
+        with open(OVOS_CONFIG) as f:
+            cfg = json.load(f)
+    except Exception as e:
+        LOG.error(e)
 
-# make holmesV Configuration.get() load neon.conf
-# TODO HolmesV does not yet support yaml configs, once it does
-#  Configuration.get() will be made to load the existing neon config files,
-#  for now it simply provides correct default values
-setup_holmes_config()
+    cfg = merge_dict(cfg, _NEON_OVOS_CONFIG)
+    with open(OVOS_CONFIG, "w") as f:
+        json.dump(cfg, f, indent=4, ensure_ascii=True)
 
 
 def setup_ovos_config():
+    """
+    Configure OVOS config to read from neon.conf files and set this path as the root.
```

Comment:
maybe clarify this method is only configuring ovos_utils

TODO: make ovos-core setup ovos_utils internally now that ovos_utils is a dep of ovos-core